From f1d4d0ff9cf4c5a6f20c000b8a1d0bc684d9e514 Mon Sep 17 00:00:00 2001
From: "Benjamin A. Beasley" <code@musicinmybrain.net>
Date: Mon, 6 Feb 2023 14:13:50 -0500
Subject: [PATCH] Fix a typo introduced in PR#501

---
 nipy/cli/tsdiffana.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/nipy/cli/tsdiffana.py b/nipy/cli/tsdiffana.py
index d4709f267..483018905 100755
--- a/nipy/cli/tsdiffana.py
+++ b/nipy/cli/tsdiffana.py
@@ -53,7 +53,7 @@
 ``/some/path/tsdiff_fname.png, /some/path/dv2_max_fname.img`` etc.
 '''
 
-from argparse ArgumentParser, RawDescriptionHelpFormatter
+from argparse import ArgumentParser, RawDescriptionHelpFormatter
 
 
 def main():
